File: FE.Core.FW/src/apis/models/toefl-challenge/ProvinceModel.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { DistrictModel } from "./DistrictModel";
import type { ProvinceType } from "./ProvinceType";

export type ProvinceModel = {
  id?: string;
  code?: string | null;
  name?: string | null;
  type?: ProvinceType;
  districts?: Array<DistrictModel> | null;
};

File: BE.Core.FW/Backend/Model/ProvinceModel.cs
namespace Backend.Model
{
    public class ProvinceModel
    {
        public Guid Id { get; set; }
        public string Code { get; set; } = default!;
        public string Name { get; set; } = default!;
        public int Area { get; set; }
        public List<DistrictModel>? Districts { get; set; }
    }
}
